src/components/KPop/KPop.vue

```diff
@@ -335,41 +337,23 @@ onMounted(() => {
       }
 
       if (props.trigger === 'hover') {
-        popoverTrigger.value.addEventListener('mouseenter', showPopover)
-        popoverTrigger.value.addEventListener('focus', showPopover)
-        popoverTrigger.value.addEventListener('mouseleave', hidePopover)
-        popoverTrigger.value.addEventListener('blur', hidePopover)
+        useEventListener(popoverTrigger.value, 'mouseenter', showPopover)
+        useEventListener(popoverTrigger.value, 'focus', showPopover)
+        useEventListener(popoverTrigger.value, 'mouseleave', hidePopover)
+        useEventListener(popoverTrigger.value, 'blur', hidePopover)
```

Comment:
```suggestion
        useEventListener(popoverTrigger.value, 'mouseenter', showPopover)
        useEventListener(popoverTrigger.value, 'focusin', showPopover)
        useEventListener(popoverTrigger.value, 'mouseleave', hidePopover)
        useEventListener(popoverTrigger.value, 'focusout', hidePopover)
```

Should this also use `focusin` and `focusout`? eg. the trigger element is not focusable itself but its descendant is. Like an `<input>` with a wrapper.

Then we can simplify the code further as:

```ts
if (props.trigger === 'hover') {
  [popoverTrigger.value, popoverElement.value].forEach((el) => {
    if (!el) {
      return
    }
    useEventListener(el, 'mouseenter', showPopover)
    useEventListener(el, 'focusin', showPopover)
    useEventListener(el, 'mouseleave', hidePopover)
    useEventListener(el, 'focusout', hidePopover)
  })
}
```

Reply:
Good call, addressed.